libs/cardano-ledger-core/src/Cardano/Ledger/DRep.hs

```diff
@@ -186,7 +186,9 @@ instance ToJSON DRepState where
           object $
             [ "expiry" .= toJSON drepExpiry
             , "deposit" .= toJSON drepDeposit
-            , "delegators" .= toJSON drepDelegs
+            -- Since the corresponding `FromJSON` instance ignores this field, we omit it from
+            -- the `ToJSON` instance as well. Round-tripping is therefore as expected.
+            -- , "delegators" .= toJSON drepDelegs
```

Comment:
Just a nitpick, feel free to ignore it - i would mention the name of the field instead of the last line, which looks a bit like dead code we forgot to remove. Maybe something like :
```suggestion
            -- Since the corresponding `FromJSON` instance ignores the `delegators` field,
            -- we omit it from the `ToJSON` instance, ensuring that round-tripping behaves as expected.
```

Reply:
Thanks for the nice suggestion! 🙌